feat: adds more informative package README (#2130)
trpc/trpc: packages/client/README.md

<p align="center">
  <a href="https://trpc.io/"><img src="../../www/static/img/logo-text.svg" alt="tRPC" height="130"/></a>
</p>

<p align="center">
  <strong>End-to-end typesafe APIs made easy</strong>
</p>

<p align="center">
  <!-- TODO: replace with new version GIF -->
  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Demo" />
</p>

# `@trpc/client`

> > The `@trpc/client` package is responsible for communicating with a tRPC server on the client side.

Reviewer: ```suggestion
> Communicate with a tRPC server on the client side.
```

## Documentation

Full documentation for `@trpc/client` can be found [here](https://trpc.io/docs/vanilla)

## Installation

```bash

> # NPM

Reviewer: ```suggestion
# npm
```

npm install @trpc/client

# Yarn
yarn add @trpc/client

> # Pnpm

Reviewer: ```suggestion
# pnpm
```

> pnpm install @trpc/client

Reviewer: ```suggestion
pnpm add @trpc/client
```

```

## Basic Example

```typescript
import { createTRPCClient, createTRPCClientProxy } from '@trpc/client';

// Importing the router type from the server file
import type { AppRouter } from './server';

// Initializing the tRPC client
const client = createTRPCClient<AppRouter>({

>   url: `http://localhost:2022`,

Reviewer: ```suggestion
  url: `http://localhost:3000`,
```
Next's default port
